examples/notebook/plots.ipynb

```diff
@@ -0,0 +1,113 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# This notebook demonstrates basic tensor operations in Burn."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "vscode": {
+     "languageId": "rust"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "// Dependency declarations for the notebook. WARNING: It may take a while to compile the first time.\n",
+    "\n",
+    "// The syntax is similar to the one used in the Cargo.toml file. Just prefix with :dep\n",
+    "// See: https://github.com/evcxr/evcxr/blob/main/COMMON.md\n",
+    "\n",
+    ":dep burn = {path = \"../../burn\"}\n",
+    ":dep burn-ndarray = {path = \"../../burn-ndarray\"}\n",
+    "\n",
+    "// The following dependencies are used for plotting\n",
+    ":dep image = \"0.23\"\n",
+    ":dep evcxr_image = \"1.1\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "vscode": {
+     "languageId": "rust"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "// Import packages\n",
+    "use burn::tensor::Tensor;\n",
+    "use burn_ndarray::NdArrayBackend;\n",
+    "\n",
+    "// Import plotting library\n",
+    "use evcxr_image::ImageDisplay;\n",
+    "\n",
+    "// Type alias for the backend\n",
+    "type B = NdArrayBackend<f32>;"
```

Comment:
I'm not sure Burn is actually used in this notebook, maybe we should remove the dependencies to make the compilation quicker?

Reply:
I want to come back and use it. I added a TODO. Lets leave it for now..